website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -154,6 +161,59 @@ The **auto_healing_policies** block supports:
 * `initial_delay_sec` - (Required) The number of seconds that the managed instance group waits before
  it applies autohealing policies to new instances or recently recreated instances. Between 0 and 3600.
 
+The **version** block supports:
+
+```hcl
+version {
+ name = "appserver-canary"
+ instance_template = "${google_compute_instance_template.appserver-canary.self_link}"
+ target_size_fixed = 1
+}
+```
+
+```hcl
+version {
+ name = "appserver-canary"
+ instance_template = "${google_compute_instance_template.appserver-canary.self_link}"
+ target_size_percent = 20
+}
+```
+## Example Usage with multiple Versions
```

Comment:
Sorry if I wasn't clear- you're more than welcome to leave this example up at the top- I just wanted the caveat about target sizes moved (which you did)

Reply:
My bad, should be fixed now!